react/src/components/atoms/forms/ErrorMessage/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SvgInputError from '../../icons/SvgInputError';
+import SvgInputSuccess from '../../icons/SvgInputSuccess';
+import './style.css';
+
+const ErrorMessage = ({inputId, message, status}) => {
+  const isSuccessful = status === 'success';
+  return(
+    <div
+      htmlFor={inputId}
+      aria-labelledby={inputId}
+      className={`ma__error-msg has-error ${isSuccessful ? 'ma__error-msg--success' : ''}`}
```

Comment:
should probably add something in style.scss for rendering the message text (font) green in a success state.